src/Symfony/Component/Form/Extension/Core/EventListener/ResizeFormListener.php

```diff
@@ -52,12 +52,12 @@ class ResizeFormListener implements EventSubscriberInterface
      */
     private $deleteEmpty;
 
-    public function __construct($type, array $options = array(), $allowAdd = false, $allowDelete = false, $deleteEmpty = false)
+    public function __construct($entryType, array $entryOptions = array(), $allowAdd = false, $allowDelete = false, $deleteEmpty = false)
```

Comment:
Can you revert the variable renames in this method? They don't really make sense now IMO.